scene/gui/graph_edit_arranger.cpp

```diff
@@ -77,15 +74,16 @@ void GraphEditArranger::arrange_nodes() {
 		if (graph_element->is_selected() || arrange_entire_graph) {
 			selected_nodes.insert(graph_element->get_name());
 			HashSet<StringName> s;
-			for (List<GraphEdit::Connection>::Element *E = connection_list.front(); E; E = E->next()) {
-				GraphNode *p_from = Object::cast_to<GraphNode>(node_names[E->get().from_node]);
-				if (E->get().to_node == graph_element->get_name() && (p_from->is_selected() || arrange_entire_graph) && E->get().to_node != E->get().from_node) {
+			List<Ref<GraphEdit::Connection>> connection_list = graph_edit->get_connection_list();
```

Comment:
I'm confused, why would we want to fetch it for every item again and again?

Reply:
I'm confused as well, it's a mistake :)